main.py

```
@@ -0,0 +1,179 @@
+
+# Obtén las variables de entorno
+PROJECT_ID = os.getenv("PROJECT_ID")
+LOCATION = os.getenv("LOCATION")
+SEARCH_ENGINE_ID = os.getenv("SEARCH_ENGINE_ID") # ID de tu motor de búsqueda de Vertex AI Search (engine_id)
+DATA_STORE_ID = os.getenv("DATA_STORE_ID") # ID de tu almacén de datos de Vertex AI Search (data_store_id)
+
+# Verifica que las variables de entorno necesarias estén configuradas
+if not all([PROJECT_ID, LOCATION, SEARCH_ENGINE_ID, DATA_STORE_ID]):
+    raise ValueError(
+        "Las variables de entorno PROJECT_ID, LOCATION, SEARCH_ENGINE_ID y DATA_STORE_ID deben estar configuradas."
+    )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)